common/pop_types/cwp_creatives.txt

```diff
@@ -7,7 +7,7 @@
 		169
 	}
 	strata = middle
-	start_quality_of_life = 8
+	start_quality_of_life = 10
```

Comment:
Creatives are amongst the poorest of the middle class barring Hollywood actors which we classify as "investors."